xml/System.Runtime.ExceptionServices/ExceptionDispatchInfo.xml

```diff
@@ -216,8 +216,8 @@
         <Parameter Name="source" Type="System.Exception" Index="0" FrameworkAlternate="netcore-2.0;netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="source">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="source">The exception whose state is captured, then thrown.</param>
+        <summary>Throws the source exception, maintaining the original bucketing details and augmenting rather than replacing the original stack trace.</summary>
```

Comment:
```suggestion
        <summary>Throws the source exception, maintaining the original Watson information and augmenting rather than replacing the original stack trace.</summary>
```
bucketing is not likely to be understood; the type remarks uses "Watson information."